src/utils/spec/extractUserNameSas9.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { extractUserNameSas9 } from '../sas9/extractUserNameSas9'
+
+describe('Extract username SAS9', () => {
+  it('should return username', () => {
+    const response = `                  "title": "Log Off SAS User One",`
+    const username = extractUserNameSas9(response)
+
+    expect(username).toEqual('sasuseone')
+  })
+
+  it('should return username with fallback regex', () => {
+    const response = `                  "title": "Logout SAS User One",`
+    const username = extractUserNameSas9(response)
+
+    expect(username).toEqual('sasuseone')
+  })
+
+  it('should return username unknown', () => {
+    const response = `                  invalid",`
+    const username = extractUserNameSas9(response)
+
+    expect(username).toEqual('unknown (error fetching username)')
```

Comment:
I think that we should log the error and return `unknown` as a fallback username.

Reply:
This is actual username that is returned: `unknown (error fetching username)`